dashboard/client/components/app.tsx

```diff
@@ -46,7 +46,7 @@ class App extends React.Component {
   };
 
   render() {
-    const homeUrl = clusterURL();
+    const homeUrl = configStore.isClusterAdmin ? clusterURL() : workloadsURL();
```

Comment:
Wasn't getting rid of `configStore.isClusterAdmin` one of the goals? Do we still need this or could/should we have some more fine grained checks if the user has access to all needed bits for cluster overview page?

Reply:
Yes, it's one of the goals but Cluster overview is basically almost empty if we always redirect there by default. It's non-goal for this PR to refactor cluster overview completely (it will require UI/UX design). That said, we can probably replace `isClusterAdmin` with something else, for example `isAllowedResource("nodes")`.

Reply:
Fixed.